app/database/default/models/global.ts

```diff
@@ -2,17 +2,26 @@
 // See LICENSE.txt for license information.
 
 import {Model} from '@nozbe/watermelondb';
+import {field, json} from '@nozbe/watermelondb/decorators';
+
 import {MM_TABLES} from '@constants/database';
-import field from '@nozbe/watermelondb/decorators/field';
-import json from '@nozbe/watermelondb/decorators/json';
 
-export default class Global extends Model {
-    static table = MM_TABLES.DEFAULT.GLOBAL
+const {GLOBAL} = MM_TABLES.DEFAULT;
 
-    @field('name') name!: string
+// TODO : add TS definitions to sanitizer function signature.
+// TODO : atm, the return type for 'value' is string[].  However, this return type can change to string/number/etc.  A broader definition will need to be applied and this return type updated accordingly.
+
+/**
+ * The Global model will act as a dictionary of name-value pairs.  The value field can be a JSON object or any other
+ * data type.  It will hold information that applies to the whole app ( e.g. sidebar settings for tablets)
+ */
+export default class Global extends Model {
+    /** table (entity name) : global */
+    static table = GLOBAL;
 
-    // TODO : add TS definitions to sanitizer function signature.
+    /** name : The label/key to use to retrieve the special 'value' */
+    @field('name') name: string | undefined;
 
-    // TODO : atm, the return type for 'value' is string[].  However, this return type can change to string/number/etc.  A broader definition will need to be applied and this return type updated accordingly.
-    @json('value', (rawJson) => rawJson) value!: string[]
+    /** value : The value part of the key-value combination */
+    @json('value', (rawJson) => rawJson) value: string[] | undefined;
```

Comment:
Change data type to 'unknown'
```suggestion
    @json('value', (rawJson) => rawJson) value: unknown;
```